.github/workflows/reusable-check-changes.yml

```diff
@@ -0,0 +1,65 @@
+name: Check For Changes
+
+on:
+  workflow_call:
+    inputs:
+      filters_path:
+        required: false
+        type: string
+      issue_comment_specific_trigger:
+        required: false
+        type: string
+      issue_comment_global_trigger:
+        required: false
+        type: string
+        default: "/test"
+    outputs:
+      tested:
+        description: "true if the tests should be executed"
+        value: ${{ jobs.check_changes.outputs.tested }}
+      head_sha:
+        description: "HEAD commit sha"
+        value: ${{ jobs.check_changes.outputs.head_sha }}
+
+jobs:
+  check_changes:
+    runs-on: ubuntu-latest
+    name: Deduce required tests from code changes
+    outputs:
+      tested: ${{ steps.tested-tree.outputs.src }}
+      head_sha: ${{ steps.pr.outputs.head }}
+    if: |
+      (github.event_name == 'issue_comment' && (
+        github.event.comment.body == ${{ inputs.issue_comment_specific_trigger }} ||
```

Comment:
This could also learn about the backport triggers and get rid of someboilerplate in the backport workflows, right?